custom_msgs/axis loc
